### ISSUE TITLE
OpenGL not rendering, and phone getting insanely hot

### ISSUE THREAD

**Issue by ega95jch** [osmandapp/OsmAnd#19951]
Issue as above. All worked fine until a few days ago. I've cleared app cache and restarted phone. Doesn't work. Deleted and reinstalled app, answering 'no' to retaining data. Still doesn't work. The only way to make the app work is to revert to Version 1 rendering engine. This is pretty unsatisfactory for a state of the art new phone.

I've attached some screenshots showing side by side differences in rendering detail. App version is 4.7.17, and phone is Samsung S23.

Thanks in anticipation of a solution.

https://photos.app.goo.gl/UxpWMJHdRXVZonvw7

**Comment by vshcherb** [comment]
It's probably related to OsmAnd Live issues. Reinstall / map redownload 1 by 1 might help

**Comment by ega95jch** [comment]
Thanks, that's much better. However, rendering is still slower than with Version 1 engine. Shouldn't it be the other way round?

I've turned off OsmAnd Live. Should I stop using it?

**Comment by yuriiurshuliak** [comment]
Yes, do not use Live Updates at this time. Please try deleting the cache files related to live updates by navigating to Menu → Maps & Resources → Local → Cache or Live Updates. Additionally, you can set the Map magnifier to 100% via Menu → Configure map → Map rendering → Map magnifier. If these steps do not resolve the issue, try resetting your profile settings by going to Menu → Settings → App profile (Browse Map) → Reset to default.

**Comment by ega95jch** [comment]
Many thanks, that's much better!

**Comment by pierreay** [comment]
Thanks! That works!
